backend/howtheyvote/api/query.py

```diff
@@ -225,7 +225,7 @@ def handle(self) -> QueryResponse[T]:
 
         # Based on the IDs fetched from the search index, fetch full records
         # from the database
-        ids = [hit["id"] for hit in res["hits"]]
+        ids = [int(hit["id"]) for hit in res["hits"]]
```

Comment:
I checked out the branch and dropped this specific commit. It still results in 500 for me, with the commit it is working. 
I am not sure how to proceed, as, if I understand you correctly, the fix for me leads to problems on your machine? Curious. I am not sure why there would be platform specific differences here? :thinking:

Reply:
> if I understand you correctly, the fix for me leads to problems on your machine?

Yes, exactly.

As a first step, it would be interesting to understand what type the items returned from Meilisearch and SQLite have in your case.

In my case, both are strings:

```py
# print([hit["id"] for hit in res["hits"]])
['169362', '169182', '169187', '169252', '169153', '168485', '168326', '168446', '168833', '168321', '169085', '168984', '169197', '169157', '168997', '168862', '168456', '168264', '168336', '168278', '168454']

# print([result.id for result in results])
['168264', '168278', '168321', '168326', '168336', '168446', '168456', '168485', '168833', '168862', '168984', '168997', '169085', '169153', '169157', '169182', '169187', '169197', '169252', '169362']
```

> I am not sure how to proceed

One option would be to simply convert both to integers (or strings, shouldn’t matter): https://github.com/HowTheyVote/howtheyvote/pull/985#discussion_r1705757612

Reply:
`ids` is a list of Strings, whereas `results` is a list of ints.

Reply:
> One option would be to simply convert both to integers (or strings, shouldn’t matter): https://github.com/HowTheyVote/howtheyvote/pull/985#discussion_r1705757612

Ah, ich stand auf dem Schlauch gestern - wenn wir **beide Stellen** ändern, geht es bei Dir auch wieder, das meinst Du, oder?

Reply:
Ja, genau